mlir/lib/Conversion/ArithToLLVM/ArithToLLVM.cpp

```diff
@@ -503,6 +505,7 @@ struct ArithToLLVMDialectInterface : public ConvertToLLVMPatternInterface {
   void populateConvertToLLVMConversionPatterns(
       ConversionTarget &target, LLVMTypeConverter &typeConverter,
       RewritePatternSet &patterns) const final {
+    arith::populateArithExpandOpsPatterns(patterns);
     arith::populateArithToLLVMConversionPatterns(typeConverter, patterns);
   }
 };
```

Comment:
This pass is kind of deprecated: we're supposed to migrate to `--convert-to-llvm`. 
I'm concerned by additions here that wouldn't carry over to `--convert-to-llvm`!

Reply:
I am not completely sure of what you mean. 
The tests `mlir/test/Conversion/ArithToLLVM/arith-to-llvm.mlir` calls both `convert-arith-to-llvm` and `--convert-to-llvm`. If either arith-to-llvm or convert-to-llvm is updated, the other must be updated as well.

Reply:
@nujaa , wouldn't adding this in "ConvertToLLVMPass.cpp" _instead_ also work?

Reply:
I'm confused because you're only changing the `ArithToLLVMConversionPass` here, so was this behavior already implemented in the `convert-to-llvm` flow and you're just catching up?

(I guess that makes my point as well somehow: we should remove the `convert-arith-to-llvm` pass: it does not provide anything that `convert-to-llvm` can't do, hopefully).

Reply:
> @nujaa , wouldn't adding this in "ConvertToLLVMPass.cpp" _instead_ also work?

I think it is Arith specific and should not be part of generic ConvertToLLVMPass

Reply:
> so was this behavior already implemented in the `convert-to-llvm` flow and you're just catching up?

Neither arith-to-llvm nor convert-to-llvm was not expanding ops.